### PR TITLE
Add task to release wdl to make a github release

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: release/release_workflow.wdl [hunk 1]
```diff
@@ -186,7 +186,48 @@ task versionPrep {
...
+        RELEASE_ID=$(cat release_response | python -c "import sys, json; print json.load(sys.stdin)['id']")
+        UPLOAD_URL=$(cat release_response | python -c "import sys, json; print json.load(sys.stdin)['upload_url']")
+        # Maybe update this when we have basename !
+        UPLOAD_URL=$(echo $UPLOAD_URL | sed 's/{.*}/?name=cromwell-${newVersion}.jar/g')
+        
+        # Upload the cromwell jar as an asset
+        curl -X POST --data-binary @${cromwellJar} -H "Authorization: token ${githubToken}" -H "Content-Type: application/octet-stream" $UPLOAD_URL
+        
+        # Publish the draft
+        curl -X PATCH -d '{"draft": false}' https://api.github.com/repos/horneth/cromwell/releases/$RELEASE_ID?access_token=${githubToken}
```
Comment (mcovarr) [inline release/release_workflow.wdl]: I think the organization might need an adjustment  😄

Comment (Horneth) [inline release/release_workflow.wdl]: Oops

**Comment by mcovarr** [comment]
:+1:

File: release/release_workflow.wdl [hunk 2]
```diff
@@ -186,7 +186,49 @@ task versionPrep {
...
 
+task makeGithubRelease {
+    String githubToken
+    String organization
+    File cromwellJar
+    Int oldVersion
+    Int newVersion
+    
+    command <<<
+        set -e
```
Comment (geoffjentry) [inline release/release_workflow.wdl]: By any chance did you run this through shellcheck? if not, can you?

Comment (Horneth) [inline release/release_workflow.wdl]: I just did and fixed it accordingly. I also tested the script on my Cromwell fork.

**Pull request merged**